package-lock.json

```diff
@@ -1123,9 +1123,9 @@
       }
     },
     "lodash": {
-      "version": "4.17.5",
-      "resolved": "https://registry.npmjs.org/lodash/-/lodash-4.17.5.tgz",
-      "integrity": "sha512-svL3uiZf1RwhH+cWrfZn3A4+U58wbP0tGVTLQPbjplZxZ8ROD9VLuNgsRniTlLe7OlSqR79RUehXgpBW/s0IQw=="
+      "version": "4.17.11",
```

Comment:
Doesn't this have a package.json change too?

Reply:
No, just a newer version of the corrupted package is available and we need to lock it

---
In reply to: [260385450](https://github.com/Microsoft/clarity-tools/pull/54#discussion_r260385450) [](ancestors = 260385450)